src/firefox/remote.js

```diff
@@ -174,8 +174,8 @@ export class RemoteFirefox {
     const addon = await this.getInstalledAddon(addonId);
     await this.checkForAddonReloading(addon);
     await this.addonRequest(addon, 'reload');
-    log.info(
-      `${(new Date()).toTimeString()}: Reloaded extension: ${addon.id}`);
+    process.stdout.write(`${'\r' +
+      'Last extension reload: '} ${(new Date()).toTimeString()}`);
```

Comment:
Hmm, in the issue description I had requested that `--verbose` mode would not rewrite the line buffer because that would obscure some debugging messages. However, I see now that getting access to the `verbose` flag from `reloadAddon()` function will be tricky.

Here is an idea for a hack that I think is ok (unless you have a better idea!):
````diff
diff --git a/src/firefox/remote.js b/src/firefox/remote.js
index 52c94a4..2dbfa59 100644
--- a/src/firefox/remote.js
+++ b/src/firefox/remote.js
@@ -176,6 +176,7 @@ export class RemoteFirefox {
     await this.addonRequest(addon, 'reload');
     process.stdout.write(`${'\r' +
       'Last extension reload: '} ${(new Date()).toTimeString()}`);
+    log.debug('\n');
   }
 }
 

````